rest/data/host.go

```diff
@@ -103,15 +103,14 @@ func (hc *DBHostConnector) NewIntentHost(ctx context.Context, options *restmodel
 	if keyVal == "" {
 		return nil, errors.New("invalid key")
 	}
-
+	if user == nil {
```

Comment:
Since all calls to `NewIntentHost` use `MustHaveUser` so the user passed in must be non-nil, do we still have to check this? If so, can we just move it to the top of the function?

Reply:
Yeah true